### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by queenypingcap**
Updates to the description of the primary features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+TiKV is a dDistributed Key-Value database which is based on the design of Google Spanner and HBase, but much simpler without dependency on any distributed file system. With the implenmentation of the Raft consensus algorithm in Rust and consensus state stored in RocksDB, it guarantees data consistency. By using placement driver to implement sharding, data migration is now automatical. The transaction model is similar to Google's Percolator, and with some performance improvements. TiKV also provides snapshot isolation (SI) and serializable snapshot isolation (SSI), and externally consistent reads and writes in distributed transactions. See [Tikv-server software stack](#tikv-server-software-stack) for more information. TiKV has the following primary features:
```
Comment (disksing) [inline README.md]: TiKV is a **distributed** Key-Value ...

**Comment by disksing** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+With carefully designed Raft groups and placement driver, TiKV excels in horizontal scalability and can easily scale up to hold 100s of TBs of data.
 
 - __Consistent distributed transactions__  
-Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions. Placement driver plugs in a set of TS system, therefore timestamp oracle will be allocated in global. When TiKV client send transaction request to TiKV server, server will check relavant meta from the pd's etcd, and then TS generate two valid timestamps in two separated handling phase implying start timestamps and commit timestamps. Before transaction ends, the start timestamp will be carrying on util the transaction is committed. Once transaction is committed, externally-consistent transactions succeed.
+Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions. 
 
 - __Coprocessor support__  
-TiKV will be accessiable and any application can develop on TiKV. We tend to provide a simple deployment solution soon after.
+You can develop any application on TiKV.
```
Comment (shenli) [inline README.md]: How about "Similar to Hbase, TiKV implements coprocessor framework to support distributed computing."?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
+TiKV is a distributed Key-Value database which is based on the design of Google Spanner and HBase, but much simpler without dependency on any distributed file system. With the implementation of the Raft consensus algorithm in Rust and consensus state stored in RocksDB, it guarantees data consistency. By using placement driver to implement sharding, data migration is now automatically. The transaction model is similar to Google's Percolator, and with some performance improvements. TiKV also provides snapshot isolation (SI) and serializable snapshot isolation (SSI), and externally consistent reads and writes in distributed transactions. See [Tikv-server software stack](#tikv-server-software-stack) for more information. TiKV has the following primary features:
```
Comment (siddontang) [inline README.md]: `is automatically` is not correct.

**Comment by ngaut** [comment]
LGTM

**Pull request merged**